Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1213,6 +1215,73 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_backgroundWebView = nil;
 }
 
+void WebExtensionContext::wakeUpBackgroundContentIfNecessary(CompletionHandler<void()>&& completionHandler)
+{
+    if (!extension().backgroundContentPath() || extension().backgroundContentIsPersistent()) {
+        completionHandler();
+        return;
+    }
+
+    if (m_backgroundWebView) {
+        bool backgroundContentIsLoading = !m_actionsToPerformAfterBackgroundContentLoads.isEmpty();
+        if (backgroundContentIsLoading)
+            queueEventToFireAfterBackgroundContentLoads(WTFMove(completionHandler));
+        else {
+            scheduleBackgroundContentToUnload();
+            completionHandler();
+        }
+
+        return;
+    }
+
+    queueEventToFireAfterBackgroundContentLoads(WTFMove(completionHandler));
+    loadBackgroundWebView();
+}
+
+void WebExtensionContext::scheduleBackgroundContentToUnload()
+{
+    if (extension().backgroundContentIsPersistent())
+        return;
+
+    ASSERT(m_backgroundWebView);
+
+    // FIXME: <https://webkit.org/b/246483> Don't unload the background page if there are open ports, pending website requests, or if an inspector window is open.
+
+    if (m_backgroundWebView.get()._isBeingInspected) {
+        RELEASE_LOG(Extensions, "Not unloading non-persistent background content for extension %{private}@ because it is being inspected.", static_cast<NSString*>(m_uniqueIdentifier));
```

Comment:
You need to put a `%@` in the log to log the identifier. Also add a space in `static_cast<NSString *>`

Reply:
Ooops, missed the `%{private}@`.